test/server-bare.test.js

```diff
@@ -145,9 +145,34 @@ describe('Session specific tests', () => {
         .insertPathCrc(db, 'PATH', 32, dbEnum.packageType.zclProperties)
         .then((pkg) => {
           packageId = pkg
+          return packageId
         })
         .then(() =>
-          queryPackage.insertSessionPackage(db, sessionId, packageId)
```

Comment:
We should rewrite these tests to use await/async instead of the old .then() patterns...

Reply:
I agree. We should probably do that for our entire code base.

Reply:
Logged https://github.com/project-chip/zap/issues/1302